CONTRIBUTING.md

```diff
@@ -75,7 +75,13 @@ changes.
 ## Nerdy Details
 ### Testing
 
-To test locally, sqlfluff uses `tox`. You can build locally using...
+To test locally, sqlfluff uses `tox`.
+
+**For Windows users**: the tox environment depends on `make` to set up the dbt test folders.
+To do that we recommend using _chocolatey_. You can find the instructions to install chocolately here: https://chocolatey.org/install.
+Once chocolatey is installed you can use `choco install make` to install `make`.
```

Comment:
If we remove the dependency on `make` via the comments I suggested in the main convo, then we can remove this.